lib/common/grpc-service.js

```diff
@@ -332,24 +364,20 @@ GrpcService.convertValue_ = function(value) {
     convertedValue = {
       structValue: GrpcService.objToStruct_(value)
     };
-  } else if (is.date(value)) {
```

Comment:
I might be missing a few pieces of the puzzle, but I guess I assumed within `decodeValue_` we would have a switch case for `timestampValue` that would create a new Date object.